datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -749,6 +811,61 @@ macro_rules! get_null_count_values {
     }};
 }
 
+// Extract the min or max value calling `func` from page idex
+macro_rules! get_min_max_values_form_page_index {
+    ($self:expr, $column:expr, $func:ident) => {{
+        if let Some((col_id_index, _field)) =
+            $self.parquet_schema.column_with_name(&$column.name)
+        {
+            if let Some(page_index) = $self.page_indexes.get(col_id_index) {
```

Comment:
It is so sad that the Page Index doesn't use the same Statistics structure as RowGroupMetadata, but I see this is how it is done in the parquet thrift specification: https://github.com/apache/parquet-format/blame/master/src/main/thrift/parquet.thrift#L938-L971  😢 



All the statistics in parquet are stored in 
https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L198-L226